src/relay/backend/vm/inline_primitives.cc

```diff
@@ -122,6 +122,7 @@ struct PrimitiveInliner : ExprMutator {
       auto global = pair.first;
       auto base_func = pair.second;
       if (auto* n = base_func.as<FunctionNode>()) {
+        if (!n->UseDefaultCompiler()) continue;
```

Comment:
Presumably all future passes which operate at the module level will also need to include a similar line to this. Would it be meaningful to mark some global functions as 'do not touch'? That might be more explicit than having to know that you shouldn't edit functions that don't use the default compiler.

Reply:
We probably cannot do that. All functions in prelude are global functions. Also lamb lifting will lift some closures to the global scope. These functions should be treated as normal ones.

Reply:
Understood.